spec/spec_helper.rb

```diff
@@ -17,3 +17,4 @@
 # These helpers are useful to have everywhere.
 require 'helpers/fake_facts'
 require 'helpers/fake_gluster'
+require 'helpers/log_matcher'
```

Comment:
We were, until there wasn't a way to make assertions about log messages. Then I may have gotten a little carried away with building (and testing) a friendly rspec matcher...